app/controllers/api/v1/pharmacy/items_controller.rb

```diff
@@ -7,7 +7,7 @@ class ItemsController < ApplicationController
         # GET /pharmacy/items[?drug_id=]
         def index
           user_program = User.current.programs.detect { |x| x["name"] == "IMMUNIZATION PROGRAM" }
-          permitted_params = params.permit(:drug_id, :current_quantity, :start_date, :end_date, :batch_number, :drug_name)
+          permitted_params = params.permit(:drug_id, :current_quantity, :start_date, :end_date, :batch_number, :drug_name, :display_details, :_drug_name)
```

Comment:
* I would recommend we require the required params for this endpoint if any.
* This endpoint is not pulling all available batches